base/ca/src/main/java/com/netscape/cmscore/dbs/CertificateRepository.java

```diff
@@ -604,49 +602,35 @@ public BigInteger getLastSerialNumberInRange(BigInteger serial_low_bound, BigInt
             logger.debug("CertificateRepository:getLastCertRecordSerialNo: returning " + ret);
             return ret;
         }
```

Comment:
Since the case when there are no records is the same to the case where the first element is outside of the range I have collapsed the two case so they are performing the same operations now.